src/lib/services/access-service.ts

```diff
@@ -217,15 +302,32 @@ export class AccessService {
         return [];
     }
 
+    async getProjectUsersForRole(
+        roleId: number,
+        projectId?: string,
+    ): Promise<IUser[]> {
+        const userIdList = await this.store.getProjectUserIdsForRole(
+            roleId,
+            projectId,
+        );
+        if (userIdList.length > 0) {
+            return this.userStore.getAllWithId(userIdList);
+        }
+        return [];
+    }
+
     // Move to project-service?
     async getProjectRoleUsers(
         projectId: string,
     ): Promise<[IRole[], IUserWithRole[]]> {
-        const roles = await this.store.getRolesForProject(projectId);
+        const roles = await this.roleStore.getProjectRoles();
 
         const users = await Promise.all(
             roles.map(async (role) => {
-                const usrs = await this.getUsersForRole(role.id);
+                const usrs = await this.getProjectUsersForRole(
```

Comment:
Do we need to abbreviate `users` to `usrs` ?

Reply:
So... when I poked at the code I saw what you saw, renamed usrs to users and then had the compiler throw a tantrum - users is defined in the higher scope. I suspect the person who originally wrote this went through the same process. While we can't change this to users, I do think we should change it to something just so we don't have a 4tth victim of this process.

...what about "projectUsers"?

Reply:
Perfect!

Reply:
Done! May we be the last people to be confused by this line of code